homeassistant/components/homeassistant/services.yaml

```diff
@@ -59,6 +59,12 @@ update_entity:
   target:
     entity: {}
 
+reload_custom_jinja:
+  name: Reload custom Jinja2 code
```

Comment:
Is "code" the right word to use?

Reply:
The Jinja documentation calls them "templates", so I'll switch to that terminology. https://jinja.palletsprojects.com/en/3.1.x/templates/#import-context-behavior